pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -535,24 +535,15 @@ def test_from_codes_with_nan_code(self):
         with pytest.raises(ValueError, match="codes need to be array-like integers"):
             Categorical.from_codes(codes, dtype=dtype)
 
-    def test_from_codes_with_float(self):
+    @pytest.mark.parametrize("codes", [[1.0, 2.0, 0], [1.1, 2.0, 0]])
+    def test_from_codes_with_float(self, codes):
         # GH21767
-        codes = [1.0, 2.0, 0]  # integer, but in float dtype
         dtype = CategoricalDtype(categories=["a", "b", "c"])
 
-        # empty codes should not raise for floats
-        Categorical.from_codes([], dtype.categories)
```

Comment:
did this get lost or moved?

Reply:
I had removed that because it seemed duplicative given that we have a `test_constructor_empty` test. Looking now I see that it isn't using `from_codes` so I can add another test for that specifically.